src/main/java/org/jabref/preferences/JabRefPreferences.java

```diff
@@ -525,8 +525,8 @@ private JabRefPreferences() {
         defaults.put(SIDE_PANE_COMPONENT_NAMES, "");
         defaults.put(SIDE_PANE_COMPONENT_PREFERRED_POSITIONS, "");
 
-        defaults.put(COLUMN_NAMES, "entrytype;author/editor;title;year;journal/booktitle;bibtexkey");
-        defaults.put(COLUMN_WIDTHS, "75;300;470;60;130;100");
+        defaults.put(COLUMN_NAMES, "groups;linked_id;field:entrytype;field:author/editor;field:title;field:year;field:journal/booktitle;field:bibtexkey");
```

Comment:
You will probably need to write a method for the migration (org.jabref.migration or similar package) to ensure that a user's selected columns are converted to the new format.